### PR TITLE
preserve reading order

### PR DESCRIPTION
**Pull request by rue-a**

### CODE REVIEW

**Comment by rue-a** [comment]
test fails bcs the target tree is not updated (does not contain readingOrder)

File: textract2page/convert_aws.py [hunk 1]
```diff
@@ -201,6 +217,21 @@ def convert_file(json_path: str, img_path: str, out_path: str) -> None:
...
+        line_region_id = f'line-region-{line_block["Id"]}'
+        pagexml_text_region_line = TextRegionType(
+        Coords=CoordsType(
+            points=points_from_awsgeometry(awsgeometry,
+                                           pil_img.width,
+                                           pil_img.height)
+        ),
+        id=line_region_id,
+        )
+        pagexml_text_region.add_TextRegion(pagexml_text_region_line)
```
Comment (bertsky) [inline textract2page/convert_aws.py]: It's of course possible in PAGE-XML to use recursive regions, but not all tools (including not all OCR-D processors) actually iterate below the top level. So at this point I'd rather add the line-regions directly to the page.

**Review by bertsky — CHANGES_REQUESTED** [review]
I recommend adding a CLI option which controls this behaviour (e.g. a flag `--line-regions` or a choice option `--dummy-region=[line,page]`).

**Comment by rue-a** [comment]
> I recommend adding a CLI option which controls this behaviour (e.g. a flag `--line-regions` or a choice option `--dummy-region=[line,page]`).

Maybe we should rather add the TextRegions for each line in any case and give an option `--preserve-textract-reading-order`, which defaults to `True`. If recursive regions are not a thing, I don't see the point in a page-encompassing region.

**Comment by bertsky** [comment]
Yes, maybe so. Recursive regions are obligatory in the presence of tables, though. When we start supporting these, we will get two additional structural levels from Textract (TABLE and CELL), corresponding to TableRegion and (recursive) TextRegion. So there will not be a need for artificial text regions inside tables. Similar for forms.

So agreed – let's just drop the page-encompassing region unconditionally.

**Pull request merged**